cxx-extras-Alpine/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM alpine
```

The CI builds for the target `deploy`, so you will need to add it to all the `Dockerfile`s.

```suggestion
FROM alpine AS deploy
```

---

Should be fixed now